plugin.php

```diff
@@ -556,6 +545,8 @@ function wpcf7_submit( $form, $result ) {
 		// JS is already doing the redirect
 		if ( isset( $_POST['_wpcf7_is_ajax_call'] ) || ! isset( $result['status'] ) ) {
 			return;
+		} elseif ( defined( 'REST_REQUEST' ) && REST_REQUEST ) {
```

Comment:
`_wpcf7_is_ajax_call` was removed in the version of CF7 that switched to using REST calls for submissions instead.